config/template.cfg

```diff
@@ -1,43 +1,25 @@
 [GAME]
-# real or sim
 type=sim
-# blue or yellow
-our_color=blue
-their_color=yellow
-#positive or negative
-our_side=negative
-# desired timestamp for the ai
-ai_timestamp=0.05
+our_color = blue
+our_side = negative
+ai_timestamp = 0.05
+play_zone = full
+autonomous_play = false
 
 [COMMUNICATION]
-# serial, sim ou disabled
-# serial when you need to control physical robots through the base-station with nrf
-# sim for simulation with grsim
-# disabled you won't send any robot commands, when you want to test and
-#   play with real vision form cameras without grsim or the base-station.
-type=sim
-
-field_port_file=config/field/sim.cfg
-
-#ui-debug thing
+type = sim
+field_port_file = config/field/sim.cfg
 ui_debug_address = 127.0.0.1
+ui_cmd_sender_port = 14444
+ui_cmd_receiver_port = 15555
 
 [IMAGE]
-kalman=true
-# 1..4
 number_of_camera = 4
-frames_to_extrapolate = 20
-
-[OUTPUT]
-#put flag to output things
 
 [STRATEGY]
-# path_part (best), astar (broken), rrt (discontinued)
-pathfinder=path_part
+pathfinder = path_part
 
 [DEBUG]
-# should always be true
-using_debug=true
-# can we modify the robots from the ui-debug, True unless in competition
-allow_debug=true
+using_debug = true
+allow_debug = true
```

Comment:
Are those still used?

Reply:
Dont know. I just wanted to clean the template